src/core/Akka/Actor/ActorBase.cs

```diff
@@ -103,6 +103,10 @@ protected ActorBase()
         {
             if (ActorCell.Current == null)
                 throw new ActorInitializationException("Do not create actors using 'new', always create them using an ActorContext/System");
+
+            if (this is IWithTimers withTimers)
+                withTimers.Timers = new Scheduler.TimerScheduler(Context);
```

Comment:
timers initialization needs to be here support scenarios, when times are used in actor constructor